js/app.js

```diff
@@ -54,180 +51,59 @@ $(document).on('ready', function() {
     return playmouth;
   }
 
-//this is where it all starts!!!!
-setUpGame();
+  //this is where it all starts!!!!
+  setUpGame();
 
-  function numSpaces(){
-    var spaces = '';
-    for( a = 0; a < boardLength; a++){
-      spaces = spaces + '/';
-    }
-    return spaces;
-  }
-
-  //truning the array of objects, into objects with different names
-  // playersArr.forEach(function (element, index){
-  //  var player+index = element;
-  // });
-
-  //sample hard coded player objects
-  //player1 = new Player("blue", "a", "count", 1);
-  //player2 = new Player("red", "b", "up", 2);
-//playersArr[0].buildRow();
-//playersArr[1].buildRow();
-//playersArr[2].buildRow();
-  //player1.buildString();
-  //player2.buildString();
-  function resetKeypress(nums){
-    for(var z = 0; z < nums; z ++){
-      playersArr[z].keyStroke(0);
-      console.log(playersArr[z]);
-    }
-    console.log(playersArr);
-  }
 
   //big constructor, to handle all functions per object/player
-  function Player(color, letter, playerPos, lpressNum) {
+  function Player(color, letter, playerPos) {
     this.playerColor = color;
-    this.playerLetter = letter;
-    //this.pressNum = Nums;
-    //building the board automatically
+    this.pressNum = 0;
```

Comment:
Decided to make pressNum an instance variable so that I could use it to reset later.